bsip-0071.md

```diff
@@ -57,22 +60,24 @@ The problem of Partial GS is that it does not resolve itself automatically when
 
 The "Prevent Global Settlement" solution adopts a new idea to handle bad debt: while bad debt appears, the smartcoin will be devaluated at a ratio of the lowest CR from among the bad debt positions, yet all the smartcoin trading features, including borrowing, margin call and force settlement, will all switch to refer to GS price to ensure the continuity and fairness of all the features.
 
+
+
 # Specifications
 
-Add one flag "Prevent Global Settlement" to each smartcoin asset.
+Add one flag "Prevent Global Settlement and External Attacks" to each smartcoin asset.
 
-Introduce new parameters: *settlement price* and *call execution price*.  They shall be calculated as:
+Introduce new parameters: *threshold price*, *settlement price* and *call execution price*. The remaining two shall be calculated as:
```

Comment:
The specifications make no mention of what is stated in the abstract: "this BSIP allows the asset owner to define an optional threshold price".  A specification should describe _who_ will set and unset the threshold price, and _when_.

Reply:
That sentence somehow got lost